tests/single.rs

```diff
@@ -50,23 +54,16 @@ describe! thunk {
         it "should drop internal data just once" {
             let counter = Arc::new(Mutex::new(0));
             let counter_clone = counter.clone();
-            let result =  Thread::scoped(move || {
+            thread::scoped(move || {
                 let value = Dropper(counter_clone);
                 let t = Thunk::<()>::new(move || {
                     // Get a reference so value is captured.
                     let _x = &value;
-
-                    panic!("Muahahahah")
```

Comment:
Yes, the panic is actually quite important, since this is testing the behavior of a thunk under a panic.